core/src/main/resources/defaults/clustertemplates/7.2.15/gcp/rt-datamart.json

```diff
@@ -35,7 +35,7 @@
               "type": "pd-ssd"
             }
           ],
-          "instanceType": "e2-standard-16"
+          "instanceType": "n1-standard-8"
```

Comment:
This can be e2-standard-8 (no need to use more expensive n1 types since we don't need ephemeral storage on the master nodes).  Can use e2-standard-8 for all master nodes for RTDM.

Reply:
Done